doc/releases/changelog-dev.md

```diff
@@ -36,7 +36,10 @@
 
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
   [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
-
+  
+* Added utility functions `operation()`, `hamiltonian()`, `pauli_sentence()` and `decompose()` to provide and 
```

Comment:
Since we are removing `qml.utils.decompose_hamiltonian`, we should add an entry to the "Breaking Changes" section and an entry to `doc/development/deprecations.rst`.

Since its a fairly obscure function not imported top-level, I'm personally OK with no deprecation cycle, but we do need to clearly communicate that is now somewhere else.

Reply:
Update docs and deprecations page 👍🏼